<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1219</width>
    <height>662</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1221</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="SYSINFO">
     <attribute name="title">
      <string>SYS INFO</string>
     </attribute>
     <widget class="QLabel" name="processor3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>380</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="boot_time">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>150</y>
        <width>491</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Boot time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="run_time">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Run time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="os_version">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>270</y>
        <width>491</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>OS version:</string>
      </property>
     </widget>
     <widget class="QLabel" name="host_name">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>401</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Host:</string>
      </property>
     </widget>
     <widget class="QLabel" name="processor4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>480</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="processor1">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>330</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="processor2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>430</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpu_version">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>330</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CPU version:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="GRAPH">
     <attribute name="title">
      <string>CPU</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CPU USAGE GRAPH</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpu_usage">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CPU usage:</string>
      </property>
     </widget>
     <widget class="QLabel" name="process_amount">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>530</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Process amount:</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpu_line">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>60</y>
        <width>1062</width>
        <height>390</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>152</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>75%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>249</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>50%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>443</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>346</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>25%</string>
      </property>
     </widget>
     <widget class="QLabel" name="threads_amount">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>530</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Threads amount:</string>
      </property>
     </widget>
     <widget class="QLabel" name="cpu_mhz">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>470</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CPU MHZ:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>MEM</string>
     </attribute>
     <widget class="QLabel" name="mem_usage">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>470</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mem usage:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>346</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>25%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MEM USAGE GRAPH</string>
      </property>
     </widget>
     <widget class="QLabel" name="mem_line">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>60</y>
        <width>1062</width>
        <height>390</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>249</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>50%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>443</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0%</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>152</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>75%</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>PROCESS INFO</string>
     </attribute>
     <widget class="QLabel" name="label_input">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>INPUT PROCESS: </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="process_input">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>400</y>
        <width>391</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QTextEdit" name="process_output">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>1171</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="check_button">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>390</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CHECK</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="process_info">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>1171</width>
        <height>311</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="process_info_name">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>1031</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PROCESSES INFO</string>
      </property>
     </widget>
     <widget class="QPushButton" name="kill_button">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>390</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>KILL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refresh_button">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>12</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
